src/NumericalAlgorithms/DiscontinuousGalerkin/LiftFlux.hpp

```diff
@@ -42,6 +42,14 @@ auto lift_flux(Variables<tmpl::list<FluxTags...>> flux,
                Scalar<DataVector> magnitude_of_face_normal) noexcept
     -> Variables<tmpl::list<db::remove_tag_prefix<FluxTags>...>> {
   auto lift_factor = std::move(get(magnitude_of_face_normal));
+  // The LGL weights are:
+  //   w_i = 2 / ((N + 1) * N * P_{N}(xi_i))
```

Comment:
I think `P_{N}(xi_i)` is supposed to be squared.

Reply:
Thanks, you're right. Good catch! I've squashed this in since it's fairly minor.